meinberlin/apps/contrib/templates/meinberlin_contrib/components/alert.html

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}
+
+<div class="alert alert--{{ alert_type }}" role="alert" aria-live="{% if alert_type == 'info' or alert_type == 'success' %}polite{% elif alert_type == 'danger' %}assertive{% endif %}" aria-atomic="true">
```

Comment:
you could simplify this with `aria-live="{% if alert_type == 'danger' %}assertive{% else %}polite{% endif %}"`